### DISCUSSION TITLE
New Requirement to change the design for MOJ

### DISCUSSION THREAD

**Discussion by WorksSmart** [TDRA-ae/aegov-dls#52] [Q&A] [answered]
Dears Design team,
Good Day
regarding the new requirement to change the design for MOJ, following the web site:

**https://designsystem.gov.ae/**
 
Kindly we need to provide us with the CSS files, Java Scripts, jQuery files, Fonts(typography) for English and Arabic and CDN that can be used for .NET web applications.  --- Our solution not Angular or React.

for more details, please feel free to call us:
050-6686019

**Comment by moiz-conceptlz** [comment, marked as answer]
Hello @WorksSmart 

The UAE design system is available as an NPM package which includes the bundled version for CSS and an optional use of the JS files. The NPM package is a plugin to TailwindCSS and you can use this to generate the output based on your code. If you follow the instructions on TailwindCSS, using postCSS, you can get an optimized version of the design system CSS based on the classes you use. 

This is a technology-agnostic package and can be used with any stack - including .NET application development. You will find instructions on how install the package on [https://designsystem.gov.ae/docs/getting-started](https://designsystem.gov.ae/docs/getting-started). 

I would also like to inform you that the use of jQuery is a choice and not mandatory - your tech stack may include this if required. 

Lastly, the usage of fonts is described in the typography guidelines on [https://designsystem.gov.ae/guidelines/typography](https://designsystem.gov.ae/guidelines/typography).
